usecases/edge.mdx

```diff
@@ -0,0 +1,44 @@
+---
+title: Edge/IoT
+description: This document describes how Flipt can be used in edge and IoT computing environments.
+mode: wide
+---
+
+## Overview
+
+Flipt is designed to be able to be used in an edge computing environment. This means that Flipt can be deployed to a device that's close to the end user. This is in contrast to a centralized computing environment where the application is deployed to a data center that's far away from the end user, such as a cloud provider.
+
+Because of Flipt's small footprint and single binary deployment, it's well suited for edge computing environments. Also, since the Flipt server is stateless, it can be deployed in a highly available configuration. This means that multiple Flipt servers can be deployed to the edge and requests can be load balanced across them, providing a highly available service to the end user.
+
+Our [Deployment](/operations/deployment) documentation provides more information on how to deploy Flipt in a highly available configuration.
+
+## Database Support
+
+Flipt's plugable data store architecture allows it to be deployed to a variety of edge computing environments without requiring traditional server-based relational databases. Specifically, Flipt's support for SQLite, [LibSQL](https://turso.tech/libsql), and [Turso](https://turso.tech/) allow it to run in environments where a traditional database is not available.
+
```

Comment:
You could additionally add that the same goes for the declarative backends. They allow you to deploy Flipt to edge locations, much like putting a CDN infront of an s3 bucket. You can deploy Flipt infront of object storage and evaluate at the edge.

Reply:
yeah i moved the section on declarative storage up and put them both under the same heading